samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_es.properties

```diff
@@ -387,9 +387,9 @@ export_as_pdf=Exportar como PDF
 export_as_xlsx=Exportar como Excel
 export_pdf=como PDF
 export_xlsx=como Excel
+export_site=Asignatura
```

Comment:
```suggestion
export_site=Sitio
```

Reply:
This is based on the example provided on the Jira. Do you want to change it anyway?
![image](https://github.com/sakaiproject/sakai/assets/53173679/8aadaeb0-767c-46a0-a0cd-c296001f6468)

Reply:
Sitio is the right label in Sakai for the ES translation, and Espai for the CA.